### PR TITLE
#162725230 Users should follow each other

### PR DESCRIPTION
**Pull request by d-rita**
**What does this PR do?**
This PR enables a logged in user to view profiles and follow/unfollow users.

**Description of tasks completed**
- Create follow, unfollow, following, followers actions
- Create the follow and profileList reducers 
- Add follow/unfollow button
- Add a profile card component
- Return a list of all profiles
- Enable follow and unfollow on multiple profiles

**How this should be manually tested**
- Fetch and check out this branch using `git fetch origin ft-follow-users-162725230 && git checkout ft-follow-users-162725230`
- Log in to Author's Haven. 
- Go to `localhost/who_to_follow` to view a list of all profiles. Click on the `follow` button to follow someone. This changes it to `following`. When `following` is clicked, it changes to `follow` to show unfollowing a user. 
- To see followers of a user, go to `localhost/profiles/username/followers` and the list of following profiles at `localhost/profiles/username/following`, where username is the name of any user. 

**Related Pivotal Tracker Story**
 [#162725230](https://www.pivotaltracker.com/story/show/162725230)

### CODE REVIEW

File: src/actions/followActions/followerAction.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      return response.json();
+    })
+    .then((data) => {
+      if (statusCode === 200) {
+        dispatch({
+          type: ActionTypes.GET_FOLLOWERS_SUCCESS,
+          payload: data.profiles,
+        });
+      } else {
+        // toast.warn('No profiles found');
```
Comment (RrNn) [inline src/actions/followActions/followerAction.js]: Commented out code here.. It needs deleting

**Comment by RrNn** [comment]
On the `/who_to_follow` page, we need to make sure that the `profile`, `create article` drop down has a higher `z-index` so that it doesnt appear behind the cards.

**Comment by RrNn** [comment]
@d-rita , the drop down still appears behind the cards.
screen shot 2019-03-07 at 11 36 07

**Comment by d-rita** [comment]
image
@RrNn it's all fixed

**Comment by RrNn** [comment]
Okay

**Review by RrNn — APPROVED** [review]
Looks good

**Pull request merged**